core/src/test/java/com/sequenceiq/cloudbreak/service/LoadBalancerConfigServiceTest.java

```diff
@@ -1065,12 +1066,43 @@ public void testCreateAzureMultipleKnoxInstanceGroups() {
     public void isdatalakeLoadBalancerEntitlementEnabled() {
         Stack azureStack = createAzureStack(StackType.DATALAKE, PRIVATE_ID_1, true);
         CloudSubnet subnet = getPrivateCloudSubnet(PRIVATE_ID_1, AZ_1);
-        DetailedEnvironmentResponse environment = createEnvironment(subnet, false, "AZURE");
+        DetailedEnvironmentResponse environment = createAzureEnvironment(subnet, false, false);
         when(blueprint.getBlueprintText()).thenReturn(getBlueprintText("input/clouderamanager-knox.bp"));
         Set<LoadBalancer> loadBalancers = underTest.setupLoadBalancers(azureStack, environment, false, true, LoadBalancerSku.BASIC);
         assertEquals(new HashSet<>(), loadBalancers);
     }
 
+    @Test
+    public void testCreateAzureLoadBalancerWithoutOutboundLoadBalancer() {
+        Stack stack = createAzureStack(StackType.DATALAKE, PRIVATE_ID_1, true);
+        CloudSubnet subnet = getPrivateCloudSubnet(PRIVATE_ID_1, AZ_1);
+        DetailedEnvironmentResponse environment = createAzureEnvironment(subnet, false, true);
+        AzureStackV4Parameters azureParameters = new AzureStackV4Parameters();
+        azureParameters.setLoadBalancerSku(LoadBalancerSku.STANDARD);
+        StackV4Request request = new StackV4Request();
+        request.setAzure(azureParameters);
+
+        when(entitlementService.azureDatalakeLoadBalancerEnabled(anyString())).thenReturn(true);
+        when(blueprint.getBlueprintText()).thenReturn(getBlueprintText("input/clouderamanager-knox.bp"));
+        when(subnetSelector.findSubnetById(any(), anyString())).thenReturn(Optional.of(subnet));
+        when(availabilitySetNameService.generateName(any(), any())).thenReturn("");
+
+        ThreadBasedUserCrnProvider.doAs(USER_CRN, () -> {
+            Set<LoadBalancer> loadBalancers = underTest.createLoadBalancers(stack, environment, request);
+
+            assertEquals(1, loadBalancers.size());
+            assert loadBalancers.stream().anyMatch(l -> LoadBalancerType.PRIVATE.equals(l.getType()));
+            assert loadBalancers.stream().noneMatch(l -> LoadBalancerType.OUTBOUND.equals(l.getType()));
```

Comment:
do not use java `assert` in tests - only use JUnit API. `assert` is a language feature only working with JVM flags `-ea` and does not provide proper unit test description

Reply:
Missed it, I was following other tests. Thank you for that. Fixed

Obs: Asserts are enabled in Intellij and our gradlew build.
Obs2: We are using it in a couple of tests (not just in this class) maybe we should check it